storage/collector.go

```diff
@@ -31,32 +32,35 @@ type Collector struct {
 func InsertCollector(newCollector Collector) error {
 	client, err := conect()
 	if err != nil {
-		return fmt.Errorf("connect error: %q", err)
+		return err
```

Comment:
Acho que vale a pena dizer que o erro é tentando conectar (que o erro já diz), porém no insert :)

Também vale a pena dar uma olhada em [Working with Errors in Go 1.13](https://blog.golang.org/go1.13-errors)

Reply:
Pensei que por descrever o erro na função interna se tornaria redundante.
Ja fiz o ajuste.